### ISSUE TITLE
Figure 1 (Reviewer 2 comment 2.1)

### ISSUE THREAD

**Issue by mostafa-razavi** [mostafa-razavi/ITIC-paper#34]
![itic-pathway-c2-4](https://user-images.githubusercontent.com/16358113/52172425-d6fdf780-273c-11e9-9300-9d334e354971.png)

2.1 Figure 1 does not show the case when saturation temperature is unknown. The revised version is indicating that the Figure 1 relates to the specific case of known saturation temperature (or accurate estimate). Adding the case of a “rare aromatic molecule” (i.e. 1-napthalenyl, 4 phenanthrenyl butane) is fine, but the initialization has benefited from earlier GEMC results from ref 62. Neither precise values of Tsat estimated nor result tables are provided for this compound. The answer is acceptable but can be improved.

Our current reponse:
> Figure 1 was modified to clearly show that the Tsat values are not known a priori, therefore the square points 11, 13, 15, 17, and 19 do not match the coexistence curve. Instead, these points deviate from the coexistence curve.

Rich's comment:
> I don’t really like the arbitrary placement above and below. I like how we had them before (all below) because I think it is more common for someone to make a uniform initial estimate that are all too low. I do think including the red points is helpful though.

Mostafa's comment:
> I think it’s important for the reader to know the flexibility we have in choosing T_est^sat. I agree that the initial estimate is likely to be uniform, but a uniform initialization might cross the coexistence curve as shown in the new figure. Maybe I should tinker with points 11, 13, 15, 17, and 19 to make it more smooth in a way that half points are above and half points are below the coexistence curve.

**Comment by ramess101** [comment]
@mostafa-razavi 

Yeah, I think it would look better if, for example, points 11, 13, 15 are below the coexistence curve (but gradually approaching it) and then 17 and 19 are above the coexistence curve.
